README.md

```diff
@@ -32,6 +33,21 @@ Run in the current repository:
 cargo build --release
 install -Dm 755 -t /usr/local/bin target/release/shh
 ```
+Additionally if you want to build and install the manpages and shell-completions
```

Comment:
I'm not sure this part is needed, it clutters the README with little benefit:
- users that install via cargo probably don't care or expect man pages and shell completions
- users that install from a distribution package already have them
- this kind of stuff is partly distribution specific, for example the paths for shell completions are different on Arch/Debian, and I don't think this README is the place to explain all that